protocol/avdl/keybase1/user.avdl

```diff
@@ -160,6 +160,7 @@ protocol user {
   // A proof the user doesn't have.
   record ProofSuggestion {
     string key;
+    string logoKey; // Logo to use, "mastodon"
```

Comment:
Gui doesn't need logoKey so it'd be nice to keep it out of the RPC interface. May work to move `.LogoKey` from `keybase1.ProofSuggestion` to `service.ProofSuggestion`